ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -0,0 +1,192 @@
+#include <wiredtiger.h>
+#include <wiredtiger_ext.h>
+#include <fstream>
+#include <list>
+#include <errno.h>
+#include <filesystem>
+#include <mutex>
+
+#include "gcp_connection.h"
+#include "gcp_log_system.h"
+#include "wt_internal.h"
+
+struct GCPStorage {
+    WT_STORAGE_SOURCE store;
+    std::list<GCPFileSystem *> gcp_fs;
+    GCP_LOG_SYSTEM *log;
```

Comment:
Let's remove this since it doesn't exist anymore.

Reply:
Did you mean remove  GCP_LOG_SYSTEM *log; ?